editors/code/package-lock.json

```diff
@@ -183,7 +183,7 @@
             "dependencies": {
```

Comment:
I don't think this file should be part of the PR?

Reply:
Certainly not. I don't know how that ended up in there. Will remove.